todo_app/templates/index.html

```diff
@@ -1,32 +1,62 @@
 {% extends "layout.html" %}
+
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">Ian M's super dooper To-Do app-a-doodle.</p>
   </div>
 
   <form method="post" >
-    <label>Task:
-      <input name="task_title" type="text" required>
-    </label>
-    <button formaction="/add" >Add</button>
+    <h2>Add</h2>
+      <ul class="list-group"> 
+        <form method="post">
+          <li class="list-group-item  bg-light d-flex">
+            <input name="task_title" type="text" class="form-control" required>
+            <button class="btn btn-default" type="submit" formaction="/add"><i class="glyphicon glyphicon-save"></i></button>
+          </li>
+        </form>
+      </ul>
   </form>
 
   <form method="post" >
-    <div class="row">
-      <div class="col-md-12">
-        <h2>Items</h2>
-          <form method="post">
-          {% for item in to_do_list %}
-              <input type="checkbox" id="task_title" name="task_title" value={{item.title}}>
-              <label >{{item.status}} - {{item.title}}</label></br>
-          {% endfor %}
-          <input type="submit" value="Complete" formaction="/complete">
-          <input type="submit" value="Delete" formaction="/delete">
-        </form>
-      </div>
-   </div>
+    <h2>To Do</h2>
+    <ul class="list-group"> 
+      {% for item in to_do_list %}
+        {% if item.status == "To Do" %}
+          <li class="list-group-item  bg-warning d-flex">
+            <input type="hidden" id="task_id" name="task_id" value={{item.id}}>
+            <button type="submit" formaction="/complete" class="btn btn-default btn-sm">
+              <span class="glyphicon glyphicon-ok"></span>
+            </button>
```

Comment:
This doesn't quite work - you end up with lots of input elements with the same name in one form so the browser sends all of them, and Flask takes the first one.

Each row in your table should be wrapped in its own `<form>` element so the task ids are separate.

Reply:
Done